### ISSUE TITLE
ImportError: cannot import name 'subset_frame' from 'great_tables._tbl_data'

### ISSUE THREAD

**Issue by kamleshmj07** [machow/reactable-py#18]
I am trying to run this example from the documentation. I am getting this error below, is this some issue? Is there a specific great tables version that needs to be installed?

```
from reactable import Reactable, embed_css
from reactable.data import cars_93

Reactable(
    cars_93.to_polars(),
    default_page_size=5,
)
```

My requirements.txt looks like below:
reactable==0.1.0
polars==1.7.0
great_tables

**Comment by machow** [comment]
Shoot sorry -- I've released the fix in v0.1.1. (the necessary functions are now vendored directly into reactable-py)

**Comment by machow** [comment]
should be resolved in v0.1.1, but definitely let me know if you run into this issue again
